execexam/advise.py

```diff
@@ -81,12 +81,10 @@ def check_advice_model(
         )
         and advice_model is None
     ):
-        return_code = 1
```

Comment:
So just curious why did you remove the return codes? Where they causing errors? I also noticed `sys.exit` is no longer in use why is that the case? I am asking because a lot of my work has been in `advise.py` and I will also have to add these changes to mine.

Reply:
We removed the exit codes because @gkapfham emphasized that there should be only one exit point for the program, which should be in `main.py`. This design choice improves debugging as it centralizes the program’s termination. By removing sys.exit from advise.py, we ensure the program only exits through main.py, making it easier to track where and why the program terminates.

Reply:
Okay that makes a lot of sense thank you!